.github/dependabot.yml

```diff
@@ -5,3 +5,7 @@ updates:
   schedule:
     interval: weekly
   open-pull-requests-limit: 10
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
I see that dependabot was set to weekly though, so should this interval be weekly as well?
https://github.com/jenkinsci/jackson2-api-plugin/blob/808e3966d6c22431c83d1aaade2ad5f582071334/.github/dependabot.yml#L6

Reply:
Nah, I think it is fine to keep the few actions we have  up to date.   They change pretty rarely.